list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative './enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
```

Comment:
- [ ] I would suggest you make the @list an empty array since in the project requirements it is required to return numbers within the array. Your code is good and this is a very small thing, so you did a great job, but please implement this small change :pray: 
![image](https://user-images.githubusercontent.com/62705947/207860713-694d3834-b354-4cc2-8849-fb69fc8b7efc.png)
![image](https://user-images.githubusercontent.com/62705947/207860879-3b39e073-f44a-4d64-9e74-d86177261cbd.png)